helpers/all.js

```diff
@@ -9,19 +9,12 @@ const _ = require('lodash');
  * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
  */
 const factory = () => {
-    return function() {
```

Comment:
I'm going to start leaving comments for each helper individually to help me keep track of what portions are approved or still under review.

Changes to this helper: approved.